client/directives/listArticles/ListArticlesDirective.js

```diff
@@ -38,6 +38,21 @@ export function ListArticlesDirective(publisher) {
                 }
             };
 
+            /**
+             * @ngdoc method
+             * @name sdListArticles#getThumbnail
+             * @param {Object} article
+             * @returns {String}
+             * @description Returns template url dependent on type
+             */
+            scope.getThumbnail = function(article) {
```

Comment:
this method is duplicated. I think you can use a service and put it there.

Reply:
good point